assets/css/amp-editor-story-blocks.css

```diff
@@ -1,14 +1,56 @@
+div[data-type="amp/amp-story-page"] {
+	border: 1px solid #ffdddd;
+}
 
-.editor-block-list__layout {
-	background-color: #ffdddd;
+div[data-type="amp/amp-story-page"],
+div[data-type="amp/amp-story-page"] .editor-inner-blocks .editor-block-list__layout {
+	min-height: 450px;
```

Comment:
On desktop an AMP story has the following style rule:

```css
[desktop]>amp-story-page {
    margin: auto!important;
    max-height: 75vh!important;
    max-width: 45vh!important;
    min-width: 320px!important;
    min-height: 533px!important;
}
```

Would that ensure that the page is displayed with the right aspect ratio and dimensions?